examples/vertical_federated_learning/README.md

```diff
@@ -0,0 +1,5 @@
+# Vertical Federated Learning
+
+## Split Learning
+### Split learning with CIFAR-10
+This [example](../cifar10/cifar10-splitnn/README.md) includes instructions on how to run [split learning](https://arxiv.org/abs/1810.06060) using the CIFAR-10 dataset and the FL simulator.
```

Comment:
Regarding the example structure,
I propose we move cifar10-splitnn into vertical federated learning, and move this under "tutorial".
so
```
examples:
    - tutorial:
        - vertical_fedearated_learning:
             - cifar10_splitnn
```


And later we might want to break cifar10 examples into examples that categorized by "algorithm" rather than "dataset".

But that is pending discussion. @ZiyueXu77 @chesterxgchen

Reply:
Sure. Let's put the new split learning example under `vertical_fedearated_learning`

Reply:
@YuanTingHsieh, moved it to tutorials.